paasta_itests/paasta_metastatus.feature

```diff
@@ -14,7 +14,7 @@ Feature: paasta_metastatus describes the state of the paasta cluster
   Scenario: High memory usage
     Given a working paasta cluster
      When an app with id "memtest" using high memory is launched
-      And a task belonging to the app with id "memtest" is in the task list
+      And a task belonging to the app with id "/memtest" is in the task list
```

Comment:
You're right to say this isn't the right place for the leading slash - though I'm not sure the step code is the right place either. We make an effort in marathon_tools[1] to hide the user from the leading slash, so I've added some logic there to be consistent.

[1] See [here](https://github.com/Yelp/paasta/blob/master/paasta_tools/marathon_tools.py#L759), [here](https://github.com/Yelp/paasta/blob/master/paasta_tools/marathon_tools.py#L883) for examples.